src/utils/makeReactNativeConfig.js

```diff
@@ -154,7 +153,7 @@ const getDefaultConfig = ({
                  * We need to use the plugin to configutr *.bundle to get minified
                  * Also disable IE8 support as we don't need it'
                  */
-                test: /\.(js|bundle)($|\?)/i,
+                test: /\.(js|(js)?bundle)($|\?)/i,
```

Comment:
Can you update the comment saying we also need to support `.jsbundle` for iOS? otherwise, it'll be confusing why the `js?` exists there